src/pypi_changes/__init__.py

```diff
@@ -15,8 +15,11 @@
 def main(args: Sequence[str] | None = None) -> int:
     options = parse_cli_arguments(args)
     distributions = collect_distributions(options)
-    info = pypi_info(distributions, options)
-    print_tree(info, options)
+    info = list(pypi_info(distributions, options))
```

Comment:
I personally would prefer to keep this an iterator, and only materialize it later where it is needed to be a list.

Reply:
Done